### ISSUE TITLE
Beast Binary Mode-A/C Format

### ISSUE THREAD

**Issue by srsampson** [vradarserver/vrs#23]
On line:

https://github.com/vradarserver/vrs/blob/master/VirtualRadar.Library/Listener/BeastMessageBytesExtractor.cs#L246

Format specifies 4 bytes for Mode-A/C code. The Original Beast Documentation says:

https://wiki.jetvision.de/wiki/Mode-S_Beast:Data_Output_Formats

```
ESC:0x31:6 byte timestamp:1 byte amplitude:2 bytes - each nibble being an octal digit
```

**Comment by vradarserver** [comment]
Thanks for letting me know about this one @srsampson, much appreciated.

The bug has been fixed in the following commits:

v2: 8d530774c3669892514c951acaadaf82b72bdb64
v3: 236c616b1eede404e62ef725dcf2add0eff26bff

The effect of the bug is that any message that follows either a mode A or mode C reply would have been ignored by VRS. There can be a lot of mode A and C messages so the effect can be a dramatic drop-off in messages decoded. In my non-scientific test using a 30 second clip of a Beast receiver recording I decoded ~1200 messages with the bug and ~1650 messages once it was fixed.
